### ISSUE TITLE
Command is failing with 1 but no output

### ISSUE THREAD

**Issue by Anahkiasen** [ahmednuaman/grunt-scss-lint#58]
Using scss-lint 0.26, if there are errors in the SCSS files the command fails with a status of 1 (`Lints with a severity of warning were reported (no errors)`) but I get no errors shown, I have to run the command myself to see what the warnings are:

```
Macedoine ➜  weholi git:(develop) ✗ scss-lint public/app/sass --config=".grunt/.scss-lint.yml"
public/app/sass/modules/_icons.scss:12 [W] UnnecessaryParentReference: Unnecessary parent selector
public/app/sass/modules/_icons.scss:17 [W] UnnecessaryParentReference: Unnecessary parent selector
public/app/sass/modules/_mediablock.scss:80 [W] UnnecessaryParentReference: Unnecessary parent selector
public/app/sass/modules/_mediablock.scss:89 [W] UnnecessaryParentReference: Unnecessary parent selector
public/app/sass/patterns/_typography.scss:96 [W] UnnecessaryParentReference: Unnecessary parent selector

Macedoine ➜  weholi git:(develop) ✗ grunt scsslint
Running "scsslint:dist" (scsslint) task
Running scss-lint on dist
>> scss-lint failed with error code: 1
>> and the following message:Error: Command failed:
Warning: Task "scsslint:dist" failed. Use --force to continue.

Aborted due to warnings.
```
